docs/en/docs/configurations/staticfiles.md

```diff
@@ -41,3 +41,12 @@ but also via settings.
 ```
 
 This will make sure you keep the settings clean, separated and without a bloated **Esmerald** instance.
+
```

Comment:
too many spaces. Remove this please. DO you run the pre-commit locally?

Reply:
thx, I thought I had added it like in edgy and co but it wasn't